puppet/lib/puppet/util/puppetdb/command.rb

```diff
@@ -58,7 +58,7 @@ def submit
 
     begin
       response = profile("Submit command HTTP post", [:puppetdb, :command, :submit]) do
-        Http.action("#{CommandsUrl}?checksum=#{checksum}") do |http_instance, path|
+        Http.action("#{CommandsUrl}?checksum=#{checksum}", :command) do |http_instance, path|
```

Comment:
seems kind of odd that our url tells us whether the request is a `:command` or a `:query` and you have to specify it as well